python/ray/dashboard/subprocesses/handle.py

```diff
@@ -201,12 +205,13 @@ async def _do_periodic_health_check(self):
         2. log the last N lines of the log file
         3. restart the module
         """
+        incarnation = self.incarnation
         while True:
             try:
                 await self._do_once_health_check()
             except Exception:
                 filename = module_logging_filename(
-                    self.module_cls.__name__, self.config.logging_filename
+                    self.module_cls.__name__, incarnation, self.config.logging_filename
```

Comment:
Let's not have the `incarnation` as part of the log name. Can always just be dashboard_xxxx.log